pkg/auth/dependency/forgotpwdemail/template.go

```diff
@@ -29,6 +29,17 @@ Thanks.`,
 var TemplatePasswordEmailHTML = template.Spec{
 	Type:   TemplateItemTypeForgotPasswordEmailHTML,
 	IsHTML: true,
+	Default: `<!DOCTYPE html>
+<html>
+<body>
+<p>Dear {{ .email }},</p>
+<p>You received this email because someone tries to reset your account password on {{ .appname }}. To reset your account password, click this link:</p>
+<p>{{ .link }}</p>
```

Comment:
{{ .link }} will put a HTMl link here? Or just the URL?

Reply:
https://github.com/SkygearIO/skygear-server/issues/1161